x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -132,7 +133,7 @@ export const SearchBar: FC<SearchBarProps> = (opts) => {
         setIsLoading(false);
 
         let aggregatedResults: GlobalSearchResult[] = [];
-        if (searchValue.length !== 0) {
+        if (searchValue.length !== 0 && searchValue.length <= maxBytes) {
```

Comment:
This isn't necessarily my specialty, but-- every character is 2 bytes, so we _might_ need to divide by two...?

Reply:
I also wouldn't consider this my speciality - changed in commit [2131f0c](https://github.com/elastic/kibana/pull/186560/commits/2131f0c0bedb858c885ee27436a8edb4ce023849) thank you!